src/main/java/com/bootcamp/demo/restapi/BookingsController.java

```diff
@@ -0,0 +1,135 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Booking;
+import com.bootcamp.demo.model.PaymentStatus;
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.model.component.Location;
+import com.bootcamp.demo.model.component.ScooterStatus;
+import com.bootcamp.demo.service.BookingService;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Logger;
+
+
+/**
+ * Sample RestController
+ * Demo - Used for testing purposes
+ */
+
+@RestController
+@RequestMapping(path = "/api/bookings")
+public class BookingsController {
+    private static final ResponseEntity<Object> SUCCESS_RESPONSE = new ResponseEntity<>(HttpStatus.OK);
+    private static final ResponseEntity<Object> FAILURE_RESPONSE = new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+    private static final Logger LOGGER = Logger.getLogger(BookingsController.class.getName());
+    private final BookingService bookingService;
+
+    public BookingsController(BookingService bookingService) {
+        this.bookingService = bookingService;
+    }
+
+    @PostMapping
+    @ResponseStatus(value = HttpStatus.CREATED)
+    public ResponseEntity<Object> createBooking(@RequestBody final Booking booking) {
```

Comment:
Since some functionality is common from this controller with what @mariatirlea did in https://github.com/nymphadora-tonks-team/booking-jb/pull/22, you should sync and see which version is needed.